### DISCUSSION TITLE
Count people in zone in real time

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1259] [Q&A] [answered]
Hi, 
Would like to know if it's possible to use the example count people but in real time with a webcam or something else. Is it  similar than the time_in_zone example ? Thank you !

**Comment by LinasKo** [comment]
Hi @Touville01 👋 

Check out this tutorial - I believe it's what you're looking for.
https://blog.roboflow.com/how-to-count-objects-in-a-zone/

**Comment by Touville01** [comment]
Thank you but it seems to be a people counter but with a mp4 file ? My question is if it's different with real time processing and anotating

**Reply by LinasKo** [reply, marked as answer]
Yup, webcams work too.

You can use [InferencePipeline](https://inference.roboflow.com/quickstart/run_model_on_rtsp_webcam/) or get frames from [OpenCV](https://docs.opencv.org/4.x/dd/d43/tutorial_py_video_display.html).

**Reply by onuralpszr** [reply]
@LinasKo  you beated me (I agreed the solutions)

**Reply by Touville01** [reply]
Thank you both! 
Also, Is there a better method? or is the difference not too noticeable?
Thanks

**Reply by LinasKo** [reply]
These are the main methods for live data as far as I know. I'd encourage you to try both and see how they perform on your use case :)
